docs/design/specs/Ecma-335-Augments.md

```diff
@@ -366,3 +366,72 @@ In addition to the guarantees that apply to all type initializers, the CLI shall
 2. A module initializer shall run exactly once for any given module unless explicitly called by user code.
 
 3. No method other than those called directly or indirectly from the module initializer will be able to access the types, methods, or data in a module before its initializer completes execution.
+
+## Static Interface Methods
+
+This WIP section is intended to specify ECMA standard changes implied by the static interface methods. For the reference ECMA spec, I have used
+
+https://www.ecma-international.org/publications-and-standards/standards/ecma-335/
+
+(6th edition, June 2012), please let me know if I should look at a different version. It also kind of sucks that we cannot reference the ECMA spec
+dynamically via some hyperlinks - I assume there's no online version of the spec that would support referring to the individual sections and such.
+
+Based on initial audit I believe changes potentially need to be made at least to the following sections:
+
+* Virtual methods (I.8.4.4, page 25)
+* Explicit method overrides (II.9.10, page 136)
+* Constraints on generic parameters (II.9.11, page 137)
+* Introducing and overriding virtual methods (II.10.3, page 147)
+* Semantics of interfaces (II.12, page 157)
+* Interface implementation examples (II.12.2.1, page 159)
+* Defining, referencing and calling methods (II.15, page 177)
+* constrained. - (prefix) invoke a member on a value of a variable type (III.2.1, page 316)
+* call - call a method (III.3.19, page 342)
+
+We could use the "interface implementation examples" section to outline some positive and negative examples that can be also used as a basis for
+testing.
+
+Some aspects that are unclear to me now:
+
+* Especially in light of the default interface work, would it make sense to allow "overriding" (implementing) a static interface method
```

Comment:
I think this was covered in https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-02-24.md#static-abstract-members-in-interfaces: "In order to scope the initial implementation, this proposal intentionally limits such members to only abstract members, not virtual members"

Reply:
Yes. Very specifically, support for default implementation is not allowed. If a static method is virtual, it must be abstract.

Reply:
Got it, thanks for the clarification.